channels/chan_usbradio.c

```diff
@@ -3701,6 +3704,8 @@ static void _menu_print(int fd, struct chan_usbradio_pvt *o)
 	ast_cli(fd, "Tx Voice Level currently set to %d\n", o->txmixaset);
 	ast_cli(fd, "Tx Tone Level currently set to %d\n", o->txctcssadj);
 	ast_cli(fd, "Rx Squelch currently set to %d\n", o->rxsquelchadj);
+	if(o->legacyaudioscaling)
+		ast_cli(fd, "legacyaudioscaling is enabled (not recommended)\n");
```

Comment:
While I agree that legacyaudioscaling should not be enabled I'm a bit hesitant to say "not recommended" (no need to scold those who have an older configuration file).

Reply:
Updated this. Yes better to keep the text as simple as possible.